packages/sanity/src/core/releases/utils/shortRelativeDate.ts

```diff
@@ -0,0 +1,36 @@
+import {formatDistanceToNowStrict} from 'date-fns'
+
+function getTimestamp(date: Date | number | string): number {
+  if (typeof date === 'string') return Date.parse(date)
+  if (date instanceof Date) return date.getTime()
+
+  return date
+}
+
+export function shortRelativeDate(date: Date | number | string): string {
```

Comment:
Appreciate this is added, but I think we could reuse `packages/sanity/src/core/components/RelativeTime.tsx`

```
<RelativeTime time={bundle.publishedAt} useTemporalPhrase />
```

But lets check with the team given it doesn't return the exact same response.